### DISCUSSION TITLE
Question about Initialization

### DISCUSSION THREAD

**Discussion by xiaoheimai95** [CURENT/andes#419] [Q&A] [answered]
Dear Dr.Cui and other experts,
I am aiming at running a case with renewable energy based on IEEE39, however it always get an issue on initialization when I change the load as it shown in the following picture.
Therefore, I have been trying to edit the limit of REECA1, but after the setting step, it still remain the value it originally has. Could you please give me some advice on it and get it run?
![IMG_8361](https://user-images.githubusercontent.com/71546641/224212190-f22cf07f-1843-49dd-ab1b-c9994fc607f5.PNG)
![IMG_8362](https://user-images.githubusercontent.com/71546641/224212375-88e23ffb-eda3-418b-a2e5-9db474352ff6.PNG)

**Comment by cuihantao** [comment, marked as answer]
Thank you for posting the discussion.

There was just one missing data when you set up the wind models. When replacing synchronous generators with REGCA1, make sure that latter has the same capacity rating `Sn`. The `Sn=100` in your case causes some per-unit values of the controllers to be well beyond typical limits. That was the case for the current limiter `REECA1.HpHL_lim`.

Here's the case with the fix. I came up with the `Sn` for REGCA1 by following that of synchronous generators.

[ieee39_add_wind.xlsx](https://github.com/CURENT/andes/files/10943557/ieee39_add_wind.xlsx)

**Reply by xiaoheimai95** [reply]
Thanks for your answer. However, when I changed the load to, for instance, 80% or 90% of the original load, I still get the 'initialization failed' problem like this:

REECA1.IpHL_lim: *not adjusted* limit <Ipmin>
   Idx     | Input  | Old Limit
-----------+--------+----------
 REECA1_10 | -5.006 | 0
